README.md

```diff
@@ -0,0 +1,134 @@
+This is a client implementation of the Apache Camel Language Server Protocol for Emacs
+
+# Camel LSP client for Emacs
+
+Code completion for Camel LS with XML LS
```

Comment:
Code completion is only a subpart of the capabilities. Maybe "Text Editing capabilities of Camel URI with Camel XML DSL."